game/graphics/opengl_renderer/sprite/GlowRenderer.h

```diff
@@ -76,7 +76,8 @@ class GlowRenderer {
 
   // max sprites should be 128 in simple sprite, plus 256 from aux = 384
   // 20 width = 20 * 20 = 400 sprites > 384.
-  // we multiply by 4 to get 16x as many max sprites (in-game the max is only 12x)
+  // we multiply by 2 to get 4x as many max sprites (in-game the max is 12x, but hopefully won't hit
+  // this limit)
```

Comment:
You didn't actually change it lol. The aux list is expanded [here](https://github.com/open-goal/jak-project/blob/master/goal_src/jak2/engine/gfx/sprite/sprite.gc#L149), I'm pretty sure you can just change the multiplier to reduce the sprite cap so you don't have to hope for anything.

Reply:
Done. There is also the `simple-sprite-system` that can add glow sprites without using the aux list, but it looks like that is unchanged so 4x should be enough.